src/main/java/com/amazon/opendistroforelasticsearch/sql/domain/SearchResult.java

```diff
@@ -135,7 +135,7 @@ private Object covenValue(Aggregation value) throws SqlParseException {
         } else if (value instanceof LongTerms) {
             return value;
         } else {
-            throw new SqlParseException("unknow this agg type " + value.getClass());
+            throw new SqlParseException("Unknown aggregation value type: " + value.getClass());
```

Comment:
`value.getClass()` may need to be replaced by `value.getClass().getSimpleName()` for example.

Reply:
Done, thanks!